src/math/trigonometry.js

```diff
@@ -208,100 +287,105 @@ p5.prototype.sin = function(angle) {
 };
 
 /**
- * Calculates the tangent of an angle. This function takes into account
- * the current <a href="#/p5/angleMode">angleMode</a>. Values are returned in the range of all real numbers.
+ * Calculates the tangent of an angle. `tan()` is useful for many geometric
+ * tasks in creative coding. The values returned oscillate between -Infinity
+ * and Infinity. `tan()` takes into account the current
+ * <a href="#/p5/angleMode">angleMode</a>.
  *
  * @method tan
- * @param  {Number} angle the angle
- * @return {Number}       the tangent of the angle
+ * @param  {Number} angle the angle.
+ * @return {Number}       the tangent of the angle.
  *
  * @example
  * <div>
  * <code>
- * let a = 0.0;
- * let inc = TWO_PI / 50.0;
- * for (let i = 0; i < 100; i = i + 2) {
- *   line(i, 50, i, 50 + tan(a) * 2.0);
- *   a = a + inc;
+ * function draw() {
+ *   let x = frameCount;
+ *   let y = 5 * tan(x * 0.1) + 50;
+ *   point(x, y);
+ *
+ *   describe('A series of curves drawn with black dots. The curves start from the top of the canvas, continue down at a slight angle, flatten out at the middle of the canvas, then continue to the bottom.');
  * }
- * describe(`vertical black lines end down and up from center to
- *   form spike pattern.`);
  * </code>
+ * </div>
  */
 p5.prototype.tan = function(angle) {
   return Math.tan(this._toRadians(angle));
 };
 
 /**
- * Converts a radian measurement to its corresponding value in degrees.
- * Radians and degrees are two ways of measuring the same thing. There are
- * 360 degrees in a circle and 2*PI radians in a circle. For example,
- * 90° = PI/2 = 1.5707964. This function does not take into account the
- * current <a href="#/p5/angleMode">angleMode()</a>.
+ * Converts an angle measurement in radians to its corresponding value in
+ * degrees. Degrees and radians are two ways of measuring the same thing.
+ * There are 360 degrees in a circle and 2 &times; &#960; radians in a circle.
+ * For example, 90° = &#960; &divide; 2 = 1.5707964. This function does not
```

Comment:
Should we do `&pi;` here to be easier for maintainers to read in the source? not sure if it's the exact same character though

Reply:
Yep!